pmda/density.py

```diff
@@ -133,14 +128,17 @@ def _conclude(self):
 
     @staticmethod
     def _reduce(res, result_single_frame):
-        """ 'append' action for a time series"""
+        """ 'accumulate' action for a time series"""
         if isinstance(res, list) and len(res) == 0:
             res = result_single_frame
         else:
-            res[0] += result_single_frame[0]
+            res += result_single_frame
         return res
 
     @staticmethod
-    def current_coordinates(atomgroup, atomselection, update_selection):
-        return atomgroup.universe.select_atoms(atomselection,
-                                               updating=update_selection).positions
+    def current_coordinates(self, atomgroup, atomselection):
```

Comment:
If you have `self` as the first arg then you cannot have `@staticmethod`. I would probably keep it a static method and explicitly pass in all arguments, including `updating`.